packages/promptl_compiler/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@latitude-data/promptl_compiler",
```

Comment:
I bit weird this name imo. Maybe something shorter without underscore. Don't know what tho

Reply:
Changed to just `promptl` 👍